packages/eslint-config-typescript/index.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
     '@typescript-eslint/no-array-constructor': 'error',
     // no-unused-vars interference fix.
     'no-unused-vars': 'off',
-    '@typescript-eslint/no-unused-vars': 'warn',
+    '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
I probably should've asked more about this change but I approved just noticing that we were matching up eslint rules to @typescript rules. However reading the comment in lines 22-25 above I realized I'm actually not totally understanding what's going on. Are there other package updates we should fix up here besides this one?